bftengine/src/bftengine/ReadOnlyReplica.cpp

```diff
@@ -31,6 +31,7 @@
 #include "util/json_output.hpp"
```

Comment:
nit: for what is the permissions change?

Reply:
These are git's default permissions, why should this file have execution permissions?